Update docs for new Kendo JS and CSS changes (#1680)
telerik/reporting-docs: embedding-reports/display-reports-in-applications/web-application/html5-report-viewer/customizing/styling-and-appearance/overview.md

The styling and appearance of the HTML5 Report Viewer are controlled entirely through a set of CSS3 styles organized in classes with predefined names.

> All viewer-related classes are supplied by a single CSS stylesheet referenced in the report viewer HTML template. The template is provided by the Reporting REST service. To force a custom template with  a custom or default CSS specify the __templateUrl__ of the HTML5 Viewer widget option. 

Reviewer: ```suggestion
All viewer-related classes are supplied by a single CSS stylesheet that is referenced in the report viewer HTML template. The template is provided by the Reporting REST service. To enforce a custom template with a custom or default CSS, specify the __templateUrl__ of the HTML5 Viewer widget option. 
```

For reference, you can use the templates located in __%programfiles(x86)%\Progress\Reporting {{site.suiteversion}}\Html5\ReportViewer\templates__:

````JavaScript
<script type="text/javascript">
	$("#reportViewer1")
	.telerik_ReportViewer({
		serviceUrl: "/api/reports/",
		templateUrl: "/custom-templates-directory/templates/telerikReportViewerTemplate-{{buildversion}}.html"
	});
</script>
````

> The template relies on the [Kendo UI SASS Themes](https://docs.telerik.com/kendo-ui/styles-and-layout/sass-themes/overview) to provide layout settings and the color schema for the HTML5 Report Viewer:

Reviewer: ```suggestion
The template relies on the [Kendo UI SassThemes](https://docs.telerik.com/kendo-ui/styles-and-layout/sass-themes/overview) to provide layout settings and the color schema for the HTML5 Report Viewer:
```

````HTML
<link href="https://kendo.cdn.telerik.com/themes/{{kendothemeversion}}/default/default-ocean-blue.css" rel="stylesheet" />
````

> The default report viewer template and stylesheet depend on CSS media queries to adapt its layout according to the device and display. For the media queries to be activated properly the browser’s viewport needs to be initialized like this:

Reviewer: ```suggestion
The default report viewer template and stylesheet depend on CSS media queries to adapt its layout according to the device and display. For the media queries to be activated properly, the browser’s viewport needs to be configured as follows:
```
